knative-operator/pkg/common/common_test.go

```diff
@@ -0,0 +1,12 @@
+package common_test
+
+import (
+	servingv1alpha1 "knative.dev/serving-operator/pkg/apis/serving/v1alpha1"
+	"testing"
+)
+
+func verifyImageOverride(t *testing.T, registry *servingv1alpha1.Registry, imageName string, expected string) {
+	if registry.Override[imageName] != expected {
+		t.Errorf("Missing queue image. Expected a map with following override in it : %v=%v, actual: %v", imageName, expected, registry.Override)
+	}
```

Comment:
nit: This feels like something we might as well just inline.

Reply:
It was like that and I felt externalizing is better as 

```
verifyIngress(t, ks, domain)
verifyImageOverride(t, &ks.Spec.Registry, "queue-proxy", image)
verifyCerts(t, ks)
```

looks better than
```
verifyIngress(t, ks, domain)
if ks.Spec.Registry.Override["foo"] != image {
	t.Errorf("Missing queue image. Expected a map with following override in it : %v=%v, actual: %v", "foo", image, ks.Spec.Registry.Override)
}
verifyCerts(t, ks)
```